### ISSUE TITLE
D3 heatmap could better show variations in temperature.

### ISSUE THREAD

**Issue by fugalfunkster** [FreeCodeCamp/FreeCodeCamp#5986]
The example currently charts the actual temperature, but the data seems to point out variations from the baseline. It might be better to set the baseline as the center of the range, and chart variations as hotter or colder. Here's an example of what I mean: http://codepen.io/fugalfunkster/full/eJvJaK/

**Comment by SaintPeter** [comment]
@mutantspore - I believe this one was yours.  Do you think that this would be a helpful change?

**Comment by mutantspore** [comment]
Yes, my original version was as @fugalfunkster has done both with using just the variance and the blue/red color scale, and it would be trivial to change it back to that. Yes, the data is a variance from the "average" baseline. I looked at it after it was done and at other representations on the web of similar data and felt that it was better displayed as the actual temperature. So as not to make it too confusing for campers. Basically they are both the same, mine has a constant added and uses a color scale more often used for temperatures.

I'd be happy to change it back if that's what's wanted.

**Comment by mutantspore** [comment]
@fugalfunkster I see that the temp legend isn't showing all the colours in the colour map (array) used. You have 11 in your colour array, and I think the full 11 are used in the graph but only 10 in the legend. ok,  the darkest red is used in the graph but not shown on the legend.

**Comment by SaintPeter** [comment]
@mutantspore You're saying you made a deliberate choice to simplify the color scale?  I don't have a strong opinion one way or that other, but we are trying to teach.  If there is a lot of added complexity to the change, then it may be better no to make it.

**Comment by mutantspore** [comment]
I have made a version along the lines of the one made by @fugalfunkster . It's not that different to code. I did make a deliberate decision, yes. I had made one showing variance and wondered if campers actually understood what they were seeing. So I made it show actual temperature. Something people are more familiar with. @QuincyLarson and I discussed this point and as the example I had shown him displayed temperature I went with that. Perhaps he has an opinion.

Right now I have the alternative coded  and I can quickly update my Pen with a changed version if required. Someone would need to fork my update and change the zipline to point at the new one.

So in terms of my effort to code .. it's done. In terms of a campers effort to code... Much the same. 

So I guess it's down to some one calling it.
